### ISSUE TITLE
Pre-built problem on linux

### ISSUE THREAD

**Issue by seanmcilvenna** [rimmartin/saxon-node#15]
Getting the following error when using pre-built on a linux environment:

```
node: symbol lookup error: /data/camara/node.js/node_modules/saxon-node/lib/binding/saxonXslt.node: undefined symbol: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEmmPKcm
```

If I CD into the node_modules/saxon-node directory and run "node-pre-gyp rebuild", it works...

At some point I was also getting errors about saxon-node not being able to find libsaxon.so (even though I had SAXON_HOME environment variable set correctly). That happened just before I removed saxon-node and re-installed as a test prior to writing this ticket.

**Comment by rimmartin** [comment]
There may be a libstdc++.so issue.  I'll try it with different computers at work.  Maybe I should install a particular g++ 5.x version for building the prebuilt

**Comment by rimmartin** [comment]
Rebuilt and published with g++ v5.2 the 64 bit linux from ubuntu 14_04
